### PR TITLE
Eventing example

### PR DESCRIPTION
**Pull request by cschubertcs**

### CODE REVIEW

File: application/src/main/java/com/sap/cloud/s4hana/examples/addressmgr/messaging/MessageConsumer.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+public class MessageConsumer implements Consumer<Stream<Message<byte[]>>>
+{
+    private static final Logger logger = CloudLoggerFactory.getLogger(MessageConsumer.class);
+
+    @Override
+    public void accept( final Stream<Message<byte[]>> messageStream )
+    {
+        messageStream.map(Message::getContent).map(String::new).forEach(logger::info);
+        // add your custom message/event handling logic here
```
Comment (sacnl) [inline application/src/main/java/com/sap/cloud/s4hana/examples/addressmgr/messaging/MessageConsumer.java]:
```suggestion
        // replace with your custom message/event handling logic
```
+ and move this up one line.

File: manifest.yml [hunk 2]
```diff
@@ -10,8 +10,18 @@ applications:
...
+         "address": "queue:BusinessPartnerQueue",
+         "reliable": true,
+         "maxMsgInFlight": 2000}
+       }}
+  services:
 #  - my-application-logs
 #  - my-xsuaa
 #  - my-destination
 #  - my-connectivity
+    - Eventing
```
Comment (sacnl) [inline manifest.yml]:
```suggestion
    - my-enterprise-messaging
```
* in order to stay consistent with other service instance names
* also adapt in line 17

File: application/src/main/java/com/sap/cloud/s4hana/examples/addressmgr/messaging/MessageConsumer.java (hunk 1 above)
Comment (cschubertcs) [reply application/src/main/java/com/sap/cloud/s4hana/examples/addressmgr/messaging/MessageConsumer.java]: Done

File: manifest.yml (hunk 2 above)
Comment (cschubertcs) [reply manifest.yml]: Done

File: application/src/main/java/com/sap/cloud/s4hana/examples/addressmgr/messaging/MessageConsumer.java (hunk 1 above)
Comment (sacnl) [reply application/src/main/java/com/sap/cloud/s4hana/examples/addressmgr/messaging/MessageConsumer.java]: Can you also change the text to `replace with your custom message/event handling logic`?

**Pull request merged**